package.json

```diff
@@ -23,7 +23,7 @@
       "url": "http://opensource.org/licenses/Apache-2.0"
     }
   ],
-  "dependencies": {
+  "peerDependencies": {
```

Comment:
please note that this behaves different in NPM older than v3. We should also keep it under `dependencies`. I added a commit to fix that.